### ISSUE TITLE
Gradient Warning

### ISSUE THREAD

**Issue by nzomi** [InternLM/InternLM-XComposer#326]
Dear all, I've encountered a problem with the warning: 'UserWarning: None of the inputs have requires_grad=True. Gradients will be None.' Has anyone else come across this issue? Should I take any action or can I ignore it? I'm uncertain if it will affect the results. Thank you!
![image](https://github.com/InternLM/InternLM-XComposer/assets/91032945/9a51390f-02b6-482f-b46f-1d23f8a76a73)

**Comment by panzhang0212** [comment]
Gradient checkpoint are used in no_grad parameters.  This will not affect your training. You can ignore it
